examples/dlint/test_plugin.js

```diff
@@ -0,0 +1,26 @@
+class Plugin extends Visitor {
+    visitImportDeclaration(e) {
+      this.addDiagnostic({
+        filename: "test.ts",
+        message: "Import found",
+        code: "",
+        range: {
+          start: {
+            line: 1,
+            col: 1,
+            bytePos: 0
+          },
+          end: {
+            line: 1,
+            col: e.span.end,
+            bytePos: 0,
+          }
+        }
+      });
```

Comment:
I suggest this method calls into Rust, it will be easier to get proper location there